Src/Support/Google.Apis.Auth/JsonWebToken.cs

```diff
@@ -120,6 +120,12 @@ public class Payload
             [JsonProperty("typ")]
             public string Type { get; set; }
 
+            /// <summary>
+            /// A space-delimited list of the permissions the application requests or <c>null</c>.
+            /// </summary>
+            [JsonProperty("scope")]
+            public string Scope { get; set; }
```

Comment:
I'm not sure about this - the class is described as "JWT Payload as specified in http://tools.ietf.org/html/draft-ietf-oauth-json-web-token-08#section-4.1." - and that doesn't include anything about the scope.

If we want to include it here (rather than in some other class) then we should adjust the documentation carefully.

Reply:
reverting, and using GoogleJsonWebSignature.Payload